libs/barista-components/sunburst-chart/src/sunburst-chart.ts

```diff
@@ -49,13 +49,10 @@ import {
   getSlices,
   getValue,
 } from './sunburst-chart.util';
-import { Platform } from '@angular/cdk/platform';
-import { DomSanitizer, SafeStyle } from '@angular/platform-browser';
-import { DtColors } from '@dynatrace/barista-components/theming';
 
-const OVERLAY_PANEL_CLASS = 'dt-sunburst-chart-overlay-panel';
-const VIEWBOX_WIDTH = 480;
-const VIEWBOX_HEIGHT = 400;
+/** Minimum width of the chart */
+const MIN_WIDTH = 480;
+const MAX_WIDTH = 800;
```

Comment:
Why do we need a max width here? The surrounding layout should limit the chart from growing.